### ISSUE TITLE
"No module named termios" error on Windows

### ISSUE THREAD

**Issue by abhiramr** [MichaelKim0407/mk-ls-git#2]
I know  this hasn't been tested for Windows and it probably isn't even designed for it. But I was curious and tried it on git-bash. It installed fine, but when I tried to run it, I got this error - 

```Traceback (most recent call last):
  File "C:\Users\aramesh\AppData\Local\Programs\Python\Python36-32\Scripts\ls-git-script.py", line 11, in <module>
    load_entry_point('mklsgit==0.1.1', 'console_scripts', 'ls-git')()
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\site-packages\pkg_resources\__init__.py", line 565, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\site-packages\pkg_resources\__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\site-packages\pkg_resources\__init__.py", line 2291, in load
    return self.resolve()
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\site-packages\pkg_resources\__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\site-packages\mklsgit.py", line 2, in <module>
    import pty
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\pty.py", line 11, in <module>
    import tty
  File "c:\users\aramesh\appdata\local\programs\python\python36-32\lib\tty.py", line 5, in <module>
    from termios import *
ModuleNotFoundError: No module named 'termios'```

**Comment by MichaelKim0407** [comment]
Thanks for your feedback.

The current release is indeed not intended for Windows. However, it could potentially run on git-bash where `ls` command is supported. I didn't think about this because I don't work on Windows and Windows cmd uses `dir` instead of `ls`.

The `pty` (which needs `termios`) module is used here to keep the colored output for `ls`, and `termios` is *nix only. I can remove this requirement on Windows, but there will be no colored output.

I'll see what I can do and hopefully I can release a new version for Windows soon.

**Comment by abhiramr** [comment]
Thanks for the prompt response!
I'll keep an eye out for the update. 
Btw, there probably Is a way to support colored output on git-bash in an alternative manner. I'll look into it and see if anything feasible turns up.

**Comment by MichaelKim0407** [comment]
This issue has been fixed as of version 0.2.

@abhiii5459 Please try it out and tell me if there are any problems.

Also I've opened issue #3 for future discussion about colored output on Windows.

**Comment by abhiramr** [comment]
@MichaelKim0407  Yup, works fine after the update!
